code/modules/clothing/under/accessories/wristwatches.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/clothing/accessory/wristwatches
+	name = "black wrist watch"
+	desc = "A simple time keeping device. Its nylon strap fits snugly on the wrist."
+	icon_state = "wristwatch_black"
+	body_location = HANDS
```

Comment:
We should give thought to making an extra slot for accessories, since there is no reason you can't wear a wristwatch with gloves. But that's outside the scope of this PR